integration/ide/bsp-server/src/BspServerTestUtil.scala

```diff
@@ -22,7 +22,7 @@ object BspServerTestUtil {
    * Set to true when running tests to update the snapshots on the disk. Do not forget to set it back to
    * false after that.
    */
-  val updateSnapshots = false
+  val updateSnapshots = true
```

Comment:
This should not be `true` in the repo, only in local dev.

Reply:
should also be fixed in `main`

Reply:
Created https://github.com/com-lihaoyi/mill/pull/5292 to avoid this in the future.